cluster/gce/gci/configure-helper.sh

```diff
@@ -585,23 +588,14 @@ function mount-master-pd {
   safe-format-and-mount "${pd_path}" "${mount_point}"
   echo "Mounted master-pd '${pd_path}' at '${mount_point}'"
 
-  # NOTE: These locations on the PD store persistent data, so to maintain
-  # upgradeability, these locations should not change.  If they do, take care
-  # to maintain a migration path from these locations to whatever new
-  # locations.
-
   # Contains all the data stored in etcd.
+  # NOTE: This location on the PD store persistent data, so to maintain
+  # upgradeability, it should not change. If it does, take care
+  # to maintain a migration path from this location to whatever new
+  # location.
   mkdir -p "${mount_point}/var/etcd"
   chmod 700 "${mount_point}/var/etcd"
   ln -s -f "${mount_point}/var/etcd" /var/etcd
-  mkdir -p /etc/srv
-  # Contains the dynamically generated apiserver auth certs and keys.
-  mkdir -p "${mount_point}/srv/kubernetes"
-  ln -s -f "${mount_point}/srv/kubernetes" /etc/srv/kubernetes
-  # Directory for kube-apiserver to store SSH key (if necessary).
-  mkdir -p "${mount_point}/srv/sshproxy"
-  ln -s -f "${mount_point}/srv/sshproxy" /etc/srv/sshproxy
-
```

Comment:
Do we also want to clean the old `${mount_point}/srv` files if they exist?

Reply:
I don't think so. If the master is downgraded to a version without this change, it would still expect the folder to be there.